CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Multiple Primary and Histology Coding Rules Version History
 
+**Changes in version 1.24**
```

Comment:
Version 1.23 has not been released yet, can you move this to the "1.23" section and remove this 1.24?  When you add an entry, you should check the VERSION file, that will tell you whether you need to add a new one or not.